src/logging.h

```diff
@@ -166,9 +166,14 @@ namespace BCLog {
 
 BCLog::Logger& LogInstance();
 
-/** Return true if log accepts specified category */
-static inline bool LogAcceptCategory(BCLog::LogFlags category)
+/** Return true if log accepts specified category, at the specified level. */
+static inline bool LogAcceptCategory(BCLog::LogFlags category, const BCLog::Level level = BCLog::Level::Debug)
```

Comment:
Wouldn't it be better to not have a default value and just inline the value once where it is needed?

Reply:
Sure, will do.